package.json

```diff
@@ -293,6 +293,7 @@
     "ssri": "^8.0.1",
     "ts-node": "^10.8.0",
```

Comment:
I left `ts-node` for `react-app-rewired.config.js`, which is the only use in web at this point